### PR TITLE
MISUV-7940-model-unit-tests

### PR DESCRIPTION
**Pull request by james-king-hmrc**
model-unit-tests

### CODE REVIEW

**Comment by ian-hmrc** [comment]
The same comments apply to this as https://github.com/hmrc/income-tax-view-change/pull/291

File: test/connectors/optout/OptOutUpdateRequestModelSpec.scala [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        }
+        """.stripMargin)
+
+    "be formatted to JSON correctly" in {
+      Json.toJson[OptOutUpdateRequest](requestObject) shouldBe requestJson
+    }
+    "be able to parse a JSON input as a string into the model" in {
+      requestJson.validate[OptOutUpdateRequest] shouldBe JsSuccess(requestObject)
+    }
+  }
```
Comment (ian-hmrc) [inline test/connectors/optout/OptOutUpdateRequestModelSpec.scala:47]: I think we just need to focus on writing the request body. Similarly, below, we just need to focus on reading the response.

Comment (james-king-hmrc) [reply test/connectors/optout/OptOutUpdateRequestModelSpec.scala:44]: done

Comment (ian-hmrc) [reply test/connectors/optout/OptOutUpdateRequestModelSpec.scala:44]: Cool. :)

**Review by ian-hmrc — APPROVED** [review]
LGTM

**Pull request merged**